assign_rights/templates/rights/manage.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<form action="." method="POST" accept-charset="utf-8">
+	{% csrf_token %}
+	{{ form.as_p }}
+      {{ rights_granted.as_p }} 
+	  {{ rights_granted.management_form }} 
```

Comment:
```suggestion
      {{ rights_granted_form.as_p }}
      {{ rights_granted_form.management_form }} 
```
Would suggest a more descriptive variable name.